pkgs/applications/misc/parsec/default.nix

```diff
@@ -0,0 +1,4 @@
+{ targetPlatform, callPackage }:
+  if targetPlatform.isDarwin
+  then callPackage ./parsec-bin-darwin.nix { }
+  else callPackage ./parsec-bin-linux.nix { }
```

Comment:
I believe using `callPackage` in this way breaks the nixpkgs override mechanisms..? Just leaving it as one file/derivation with a `src = if stdenv.isDarwin then x else y;` (and `version`) should be fine, though there may be other approaches.

Reply:
Edited it to put the info on the if then statement